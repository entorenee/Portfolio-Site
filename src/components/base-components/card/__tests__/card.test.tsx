import React from 'react'
import { render } from '@testing-library/react'

import Card from '..'

describe('<Card />', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Card>
        <div>Hello World</div>
      </Card>,
    )
    expect(container.firstChild).toMatchSnapshot()
  })
})
