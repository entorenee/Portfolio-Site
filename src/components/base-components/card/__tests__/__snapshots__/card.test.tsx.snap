// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders correctly 1`] = `
.emotion-0 {
  margin-bottom: 1.5rem;
  border: 1px solid #ccc;
  padding: 0.5rem 0.25rem;
  box-shadow: 3px 3px 5px #656E9A;
}

<div
  class=" emotion-0"
>
  <div>
    Hello World
  </div>
</div>
`;
